helm/idol-nifi/templates/nifi-registry/nifi-registry.yaml

```diff
@@ -41,7 +42,7 @@ spec:
       imagePullSecrets:
       - name: dockerhub-secret
```

Comment:
Existing issue: this should source from `global.imagePullSecrets` rather than being hardcoded.